lectures/equalizing_difference.md

```diff
@@ -488,3 +408,100 @@ Let's compute $\frac{\partial \phi}{\partial R}$ and evaluate it numerically at
 ```
 
 We find that raising the gross interest rate $R$ increases the initial college wage premium $\phi$, in line with our earlier graphical analysis.
+
+
+## Exercise
```

Comment:
```suggestion
## Exercises
```

Reply:
Thank @mmcky for reviewing. I finished all the suggestions in #385.